src/DashboardService.Mappers/Db/DbGroupMapper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using LT.DigitalOffice.DashboardService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.Group;
+using Microsoft.AspNetCore.Http.HttpResults;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Mappers.Db;
+
+public class DbGroupMapper : IDbGroupMapper
+{
+  public DbGroup Map(CreateGroupRequest request)
+  {
+    return request is null
```

Comment:
check for null is redundant, this is already checked in the controller

Reply:
++